.github/workflows/conformance-e2e.yaml

```diff
@@ -74,7 +74,9 @@ jobs:
       max-parallel: 16
       matrix:
         include:
-          # See https://github.com/cilium/cilium/issues/20606 for configuration table
+          # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+          # ! NOTE: keep tests-e2e-upgrade.yaml config in sync !
```

Comment:
I don't suppose there would be a way to put the description of the jobs into a separate file and to pull them in the two workflows? (I'm not aware of this being able with GHA, asking just in case you know of something).

Reply:
Or a CI check that parse the YAML and check that the two matrix.include are identical. However big you make this comment (:sweat_smile:), the list is long enough that people will add/change the end without noticing the comment.

Reply:
Unfortunately, IRL it's impossible to keep them in sync :-( E.g., https://github.com/cilium/cilium/pull/29073/files#diff-6d64c043690537c26ae80866ccf018a7e9f0b2cdd9b1cffe5c40e6db3ae27c6cR129